dist/gcc-compatible/Makefile

```diff
@@ -71,9 +70,13 @@ ifneq (,$(wildcard EverCrypt_BCrypt.h))
   SOURCES	+= evercrypt_bcrypt.c
 endif
 
+-include Makefile.config
+
+ifeq (,$(ARM_CROSS))
 OBJS 		+= $(patsubst %.S,%.o,$(wildcard *-$(MARCH)$(VARIANT).S))
+USER_CFLAGS=-march=native -mtune=native -Wno-unused
```

Comment:
I'm not entirely sure how this is supposed to work.
If I understand this Makefile snipped correctly, you want to set `USER_CFLAGS += -march=native -mtune=native` if `mtune` and `march` aren't set already in `USER_CFLAGS`.
But we don't set them for cross compilation in many cases. Then this will fail.